tests/test_cli.py

```diff
@@ -540,6 +540,106 @@ def mocked_get(url, **options):
                 'hashin==0.11 \\'
             )
 
+    @cleanup_tmpdir('hashin*')
+    @mock.patch('hashin.urlopen')
+    def test_run_pep_0496(self, murlopen):
+        """Properly pass through PEP-0496 specifiers."""
```

Comment:
Please spell out a quick summary of what it is so you don't have to Google it up. 
E.g. 
```python
"""Properly pass through PEP-0496 specifiers. 
Essentially, environment markers along with the package name and version. For
example "enum==1.1.6; python_version <= '3.4'".
See https://www.python.org/dev/peps/pep-0496/
"""
```